assets/components/gridImage/gridImage.jsx

```diff
@@ -16,10 +18,11 @@ const MIN_IMG_WIDTH = 300;
 // ----- Types ----- //
 
 export type GridImg = {
-  gridId: string,
+  gridId: ImageId,
   srcSizes: number[],
   sizes: string,
   altText: ?string,
```

Comment:
Probably a future refactor, but from an accessibility point of view, this shouldn't be a `null` for any image.